CHANGELOG.md

```diff
@@ -13,6 +13,8 @@ Possible log types:
 
 ## [Unreleased]
 
+- `[added]` `is_ok` and `is_err` type guard functions. A faster alternative to `isinstance` checks (#69)
```

Comment:
Not sure about the wording here. Perhaps we can leave off the `faster`?

How about,
```suggestion
- `[added]` `is_ok` and `is_err` type guard functions as alternatives to `isinstance` (#69)
```